gradle/versions.gradle

```diff
@@ -1,6 +1,6 @@
 ext {
     gradleVersion = '6.6'
-    squitVersion = '3.0.2'
+    squitVersion = '3.0.3'
```

Comment:
Please do not update the version number since this is tied to a README update and a release.